indico/modules/events/editing/operations.py

```diff
@@ -90,12 +90,21 @@ def create_new_editable(contrib, type_, submitter, files, initial_state=InitialR
 
 
 def delete_editable(editable):
+    print('1')
     db.session.expire(editable)
+    print('2')
     for revision in editable.revisions:
+        print('3')
+        print(revision)
+        print(revision.files)
         for ef in revision.files:
+            print('4')
             ef.file.claimed = False
```

Comment:
That's a bad idea. It will result in permanent deletion of the file. Files should remain claimed even when linked to something that's soft-deleted.